1_OCaml/TP/TP1.ipynb

```diff
@@ -28,6 +28,105 @@
     "Puis tester ces fonctions sur des arguments de votre choix."
```

Comment:
<p>Au lieu de </p><pre class="ql-syntax" spellcheck="false">let coord = (x, y) in
&nbsp; &nbsp; coord ;;
</pre><p>Tu peux écrire simplement :</p><pre class="ql-syntax" spellcheck="false">(x, y)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mp2i-fsm/mp2i-2021/pull/35/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>